x/zkism/keeper/keeper.go

```diff
@@ -102,38 +102,9 @@ func (k *Keeper) Verify(ctx context.Context, ismId util.HexAddress, _ []byte, me
 	return authorized, nil
 }
 
-func (k *Keeper) validatePublicValues(ctx context.Context, ism types.ZKExecutionISM, publicValues types.EvExecutionPublicValues) error {
```

Comment:
Nice! 🙏🏻 

Context for the reader: this is where we see the primary benefit to this PR, where we assert the trusted opaque state bytes are part of our proof values. This allows to us accept the new state bytes once the proof has been verified.